src/merchantPayment/FeeContainer.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { render } from '@testing-library/react-native'
+import BigNumber from 'bignumber.js'
+import * as React from 'react'
+import 'react-native'
+import { Provider } from 'react-redux'
+import FeeContainer from 'src/merchantPayment/FeeContainer'
+import { createMockStore } from 'test/utils'
+
+describe('FeeContainer', () => {
+  it('renders correctly', () => {
+    const tree = render(
+      <Provider store={createMockStore({})}>
+        <FeeContainer amount={new BigNumber(1337)} />
+      </Provider>
+    )
+    expect(tree).toMatchSnapshot()
```

Comment:
sorry this was probably there in the previous implementation but can we avoid adding new snapshot tests and test for the expected text etc instead? i think we discussed this some time ago, snapshots are quick to write but difficult to read / review the changes later on and serve as worse documentation about what the component is supposed to do